automatic/apache-httpd/apache-httpd.nuspec

```diff
@@ -37,7 +37,7 @@ Apache will be installed as a service under the default name 'Apache'
 </description>
     <tags>apache httpd webserver admin</tags>
     <dependencies>
-       <dependency id="vcredist2015" />
+       <dependency id="vcredist2012" />
```

Comment:
Apache httpd-2.4.25 has vc14, vc11, and vc9 versions. Since vc9 is eol in August 2017 I suggest moving this to be dependent on VC14.

best case scenario would be to be dependent on VC14 with the option to use VC11 (if multiple dependency options is possible)